.github/CHANGELOG.md

```diff
@@ -31,6 +32,19 @@
   plt.hist(samples, bins=100)
   plt.show()
   ```
+
+* Adds the `sf.ops.GKP` operation, which allows the Gottesman-Kitaev-Preskill
+  state to be initialized on both the Bosonic and Fock backends.
+  [(#553)](https://github.com/XanaduAI/strawberryfields/pull/553)
+  [(#546)](https://github.com/XanaduAI/strawberryfields/pull/546)
+
+  GKP states are qubits, with the qubit state defined by:
+
+  .. math::
+  
+      \ket{\psi}_{gkp} = \cos\frac{\theta}{2}\ket{0}_{gkp} + e^{-i\phi}\sin\frac{\theta}{2}\ket{1}_{gkp}
```

Comment:
Not completely sure, if this renders well :thinking: (screenshot is from the release notes of the built docs).
![image](https://user-images.githubusercontent.com/24476053/115080863-67195900-9ed1-11eb-8274-571146c6ba2a.png)

Reply:
Good catch! I've just pushed a commit that hopefully fixes it.

Reply:
I have escaped the underscores, so this is now fixed 😆

Reply:
Ohh, just for my own understanding, how come they need to be escaped? Indeed, it looks great now! :blush:

Reply:
In order for this markdown file to display in the docs, `m2r2` converts it into ReST. Unfortunately, it seems to be interpreting the `_` as markdown syntax for italics! So it is converting them into `*`, the ReST syntax for italics.